packages/fuel-gauge/src/predicate/utils/predicate/processPredicateData.ts

```diff
@@ -0,0 +1,67 @@
+import { FuelError, ErrorCode } from '@fuel-ts/errors';
```

Comment:
I believe these test helpers can be removed, as we can create new predicate instances whenever we need to validate that bytes are being processed consistently.